solana-programs/anchor/audius-data/tests/audius-data.ts

```diff
@@ -1115,8 +1027,158 @@ describe("audius-data", function () {
+        trackOwnerPDA: delegate.userAccountPDA,
+        userAuthorityDelegateAccountPDA: delegate.userAuthorityDelegatePDA,
+        authorityDelegationStatusAccountPDA:
+          delegate.authorityDelegationStatusPDA,
+      })
+    ).to.be.rejectedWith(Error);
+
+    await expect(
+      testCreateTrack({
+        provider,
+        program: anchor.web3.Keypair.generate().publicKey, // wrong program
+        id: trackID,
+        baseAuthorityAccount: delegate.baseAuthorityAccount,
+        handleBytesArray: delegate.handleBytesArray,
+        adminStorageAccount: adminStorageKeypair.publicKey,
+        bumpSeed: delegate.userBumpSeed,
+        trackMetadata,
+        userAuthorityKeypair: delegate.userAuthorityDelegateKeypair, // substitute delegate
+        trackOwnerPDA: delegate.userAccountPDA,
+        userAuthorityDelegateAccountPDA: delegate.userAuthorityDelegatePDA,
+        authorityDelegationStatusAccountPDA:
+          delegate.authorityDelegationStatusPDA,
+      })
+    ).to.be.rejectedWith(Error);
+
+    // use different user authority delegate PDA
+    const badDelegate = await testCreateUserDelegate({
+      adminKeypair,
+      adminStorageKeypair: adminStorageKeypair,
+      program,
+      provider,
+      ...getURSMParams(),
+    });
+
+    await expect(
+      testCreateTrack({
+        provider,
+        program: anchor.web3.Keypair.generate().publicKey, // wrong program
+        id: trackID,
+        baseAuthorityAccount: delegate.baseAuthorityAccount,
+        handleBytesArray: delegate.handleBytesArray,
+        adminStorageAccount: adminStorageKeypair.publicKey,
+        bumpSeed: delegate.userBumpSeed,
+        trackMetadata,
+        userAuthorityKeypair: delegate.userAuthorityDelegateKeypair, // substitute delegate
+        trackOwnerPDA: delegate.userAccountPDA,
+        userAuthorityDelegateAccountPDA: badDelegate.userAuthorityDelegatePDA, // mismatched PDA
+        authorityDelegationStatusAccountPDA:
+          delegate.authorityDelegationStatusPDA,
+      })
+    ).to.be.rejectedWith(Error);
+
+    // use different authority delegation status PDA
+
+    await expect(
+      testCreateTrack({
+        provider,
+        program: anchor.web3.Keypair.generate().publicKey, // wrong program
+        id: trackID,
+        baseAuthorityAccount: delegate.baseAuthorityAccount,
+        handleBytesArray: delegate.handleBytesArray,
+        adminStorageAccount: adminStorageKeypair.publicKey,
+        bumpSeed: delegate.userBumpSeed,
+        trackMetadata,
+        userAuthorityKeypair: delegate.userAuthorityDelegateKeypair, // substitute delegate
+        trackOwnerPDA: delegate.userAccountPDA,
+        userAuthorityDelegateAccountPDA: delegate.userAuthorityDelegatePDA,
+        authorityDelegationStatusAccountPDA:
+          badDelegate.authorityDelegationStatusPDA,
+      })
+    ).to.be.rejectedWith(Error);
+
+    await testCreateTrack({
```

Comment:
success case should be a separate test case.

Reply:
same comment above